README.md

```diff
@@ -34,23 +34,25 @@ Create an `<auto-complete>` element with a child `input` to use as the data-entr
   querymin="2"
   optionmax="50"
   valid="please select a valid country"
+  storekeys="key1,key2"
 >
   <input type="text" id="country" name="country" size="50" required />
 </auto-complete>
 ```
 
 `<auto-complete>` attributes:
 
-|attribute|description|
-|-:|-|
-|`id`|optional ID (only necessary when two or more controls could [auto-fill an input](#auto-fill-other-inputs))|
-|`api`|REST URL (required)|
-|`resultdata`|the name of the property containing a result array of objects in the returned API JSON (not required if only results are returned)|
-|`resultname`|the name of the property in each result object which matches the search input and is used for datalist `<option>` elements (optional)|
-|`querymin`|the minimum number of characters to enter before a search occurs (default: 1)|
-|`inputdelay`|the minimum time to wait in milliseconds between keypresses before a search occurs (default debounce: 300)|
-|`optionmax`|the maximum number of auto-complete options to show (default: 20)|
-|`valid`|if set, this error message is shown when an invalid value is selected|
+|    attribute | description                                                                                                                           |
+| -----------: | ------------------------------------------------------------------------------------------------------------------------------------- |
+|         `id` | optional ID (only necessary when two or more controls could [auto-fill an input](#auto-fill-other-inputs))                            |
+|        `api` | REST URL (required)                                                                                                                   |
+| `resultdata` | the name of the property containing a result array of objects in the returned API JSON (not required if only results are returned)    |
+| `resultname` | the name of the property in each result object which matches the search input and is used for datalist `<option>` elements (optional) |
+|   `querymin` | the minimum number of characters to enter before a search occurs (default: 1)                                                         |
+| `inputdelay` | the minimum time to wait in milliseconds between keypresses before a search occurs (default debounce: 300)                            |
+|  `optionmax` | the maximum number of auto-complete options to show (default: 20)                                                                     |
+|      `valid` | if set, this error message is shown when an invalid value is selected                                                                 |
+|  `storekeys` | if set, these datadfields are addet as custom attributes in the \<options\> of the datalist                                           |
```

Comment:
Small typos: `these datafields are added`

Perhaps better:
if set to a comma-delimited list of result values, e.g. `storekeys="id,name"`, the `<options>` element has associated `data-` attributes set which can be accessed by the [`dataset` property](https://developer.mozilla.org/docs/Web/API/HTMLElement/dataset).

Reply:
i didn't know this feature - cool